docs/tutorials/mongoid-documents.txt

```diff
@@ -176,6 +177,31 @@ Types that are not supported as dynamic attributes since they cannot be cast are
 - ``DateTime``
 - ``Range``
 
+The StringifiedSymbol Type
+--------------------------
+
+This type allows users to write Ruby code that makes use of the built-in Ruby Symbol type,
```

Comment:
I would just say "This type allows users to make use of the built-in..." (since it's assumed that anybody using Mongoid is writing Ruby code heh)